releasenotes/notes/flask_improve_resource_naming_when_error_occurs-ea094145551e4ec7.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Improve resource naming of Flask request span when errors occur in middleware.
```

Comment:
This + PR title should be worded as a fix rather than an improvement.

e.g. "Fix missing Flask endpoint/route information when an exception occurs in before request hooks" ("ish")